app/models/object_store/item.rb

```diff
@@ -0,0 +1,33 @@
+module ObjectStore
+  class Item < Edition
```

Comment:
I'm not sure yet what we need from Edition but given we said earlier we don't care about a lot of stuff it adds, do we need inheritance here or should we use composition?

```ruby
module LiteEdition
  def self.included(base)
    base.class_eval do
      store_accessor :details, :field1, :field2
      …

      validates :field1, presence: true
      …
    end
  end
end

class Item < ApplicationRecord
  include LiteEdition
end
```

Reply:
I have been considering what a more composition based approach might look like in https://github.com/alphagov/whitehall/blob/edition-refactoring/docs/adr/0003-introduce-reusable-content-items.md, which might be of interest. A `LiteEdition` module might be an interesting step in that direction, to see how much behaviour we can lift off of the edition model.